multi-node-testkit/src/main/scala/org/apache/pekko/remote/testconductor/RemoteConnection.scala

```diff
@@ -56,8 +56,15 @@ private[pekko] class ProtobufEncoder extends MessageToMessageEncoder[Message] {
 private[pekko] class ProtobufDecoder(prototype: Message) extends MessageToMessageDecoder[ByteBuf] {
 
   override def decode(ctx: ChannelHandlerContext, msg: ByteBuf, out: java.util.List[AnyRef]): Unit = {
-    val bytes = ByteBufUtil.getBytes(msg)
-    out.add(prototype.getParserForType.parseFrom(bytes))
+    try {
+      val bytes = ByteBufUtil.getBytes(msg)
+      out.add(prototype.getParserForType.parseFrom(bytes))
+    } catch {
+      case NonFatal(e) =>
+        throw new RuntimeException("Error while decoding protobuf message", e)
```

Comment:
why not rethrow the e? I dislike using the super generic RuntimeException here. If we need to wrap the exception, can't we find something more specific or create a new class like NettyDecodeException.

Reply:
Can you suggest to Flink team to revert their upgrade to Pekko 1.1 if you think that this is definietly broken.

Reply:
OK, I will change it then .

Reply:
@pjfanning I have notified with an Email.